app/server/api/disbursement_api.py

```diff
@@ -165,26 +162,13 @@ def post(self):
 class DisbursementAPI(MethodView):
     @requires_auth(allowed_roles={'ADMIN': 'admin'})
     def get(self, disbursement_id):
-        accounts = db.session.query(TransferAccount)\
```

Comment:
There's no reason to send accounts, and it's very slow. No FE uses this API, so it's safe to nix